api/server.go

```diff
@@ -273,6 +275,7 @@ func (s *server) txpoolParentsHandler(jc jape.Context) {
 
 func (s *server) txpoolTransactionsHandler(jc jape.Context) {
 	jc.Encode(TxpoolTransactionsResponse{
+		Basis:          s.cm.Tip(),
```

Comment:
Do we still want to add the basis to the response here? I checked and we're never using it, so I'm assuming it may have been added at one point for testing, then refactored and overlooked.

Reply:
Technically no for V1, but `Client.TxpoolBroadcast` has the basis as its first argument. Good to be consistent and provide the user a value to put there.

Reply:
Absolutely. The basis is very important for broadcasting v2 transactions.